cfn-resources/autogen/schema-gen/README.md

```diff
@@ -0,0 +1,3 @@
+#Read OpenAPI and generate CFN Schema
+
```

Comment:
this swagger.json file is as of today, but what happens if source file gets updated? will schema-gen break if this sagger.json file gets refreshed?